disnake/permissions.py

```diff
@@ -800,19 +820,28 @@ def manage_webhooks(self) -> int:
         """:class:`bool`: Returns ``True`` if a user can create, edit, or delete webhooks."""
         return 1 << 29
 
-    @flag_value
+    @make_permission_alias("manage_guild_expressions")
     def manage_emojis(self) -> int:
-        """:class:`bool`: Returns ``True`` if a user can create, edit, or delete emojis."""
+        """:class:`bool`: An alias for :attr:`manage_guild_expressions`."""
         return 1 << 30
 
-    @make_permission_alias("manage_emojis")
+    @make_permission_alias("manage_guild_expressions")
     def manage_emojis_and_stickers(self) -> int:
-        """:class:`bool`: An alias for :attr:`manage_emojis`.
+        """:class:`bool`: An alias for :attr:`manage_guild_expressions`.
 
         .. versionadded:: 2.0
         """
         return 1 << 30
 
+    @flag_value
```

Comment:
nit: this should be before manage_emojis

Reply:
https://github.com/DisnakeDev/disnake/pull/989/commits/4ac28beea798a92db629ad2b384ea5cf8a16ffe4